# Prediction of Sales of cold drinks

We are required to predict the sales of six cold drink brands.

## Data 

Data of sales for July 2021 is given. We need to predict the day-wise sales for Augest 2021.

In [1]:
# Load libraries
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns

In [2]:
data = pd.read_csv('SalesDataProductWise.xlsx-Sheet1.csv')
data.head()

Transaction_date    Brand Name  Sales Value
0       2018-01-01     Coca Cola      3480.30
1       2018-01-01      Thums Up       991.30
2       2018-01-01       Mirinda      4169.45
3       2018-01-01         Fanta       726.75
4       2018-01-01  Mountain Dew       484.25

In [3]:
data.nunique()

Transaction_date    1287
Brand Name             6
Sales Value         6654
dtype: int64

Data has 3 columns: **'Transaction_date', 'Brand Name', 'Sales Value'**

Total no of sales: 6654

Number of Brands: 6 

In [4]:
data.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 7418 entries, 0 to 7417
Data columns (total 3 columns):
 #   Column            Non-Null Count  Dtype  
---  ------            --------------  -----  
 0   Transaction_date  7418 non-null   object 
 1   Brand Name        7418 non-null   object 
 2   Sales Value       7418 non-null   float64
dtypes: float64(1), object(2)
memory usage: 174.0+ KB


In [5]:
data.describe()

Sales Value
count    7418.000000
mean     4860.019402
std      9004.438471
min         0.200000
25%       762.525000
50%      2030.175000
75%      5702.216250
max    282293.280000